models/Campground.js

```diff
@@ -43,6 +43,11 @@ const CampgroundSchema = new mongoose.Schema(
       type: mongoose.Schema.Types.Decimal128,
       required: [true, "Please add a price"],
     },
+
+    promptpayTel: {
+      type: string,
```

Comment:
Please make it to be "String"

Reply:
@jeeholy have you tested anything before requesting a pull request?

I would like to make a clarification here if it's not yet clear for anyone. Only make a pull request when you think the branch is ready to merge. If changes are requested or some minor mistake are overlooked then it's okay. What pull request is **not**__ for is getting other to do the work for you while you haven't done any testing or checking at all.

While it's not explicitly stated, the code on PR are supposed to the very least "seem to work" and if anyone (including me) try to open a PR while the code is obviously not ready, doesn't work, or seem to be carelessly written, then other team member might felt that the person making the PR does not care about contributing to the group work. in the absolute worst case that might result in you getting a 10 points reduction from a score that everyone in the group get (meaning that person will get 10 less point than other people in the group). So let's check our work at least a little bit before opening a pull request.

With respect.